File: cpp/bc1165.cpp
#include <iostream>

using namespace std;

int main()
{
	int count, val;
	
	cin >> count;
	
	for(int i = 0; i < count; i++)
	{
		int cNu = 0;
		
		cin >> val;
		
		for(int j = 1; j <= val; j++)
			if(val % j == 0) cNu++;
		
		cNu == 2? cout << val << " eh primo" << endl : cout << val << " nao eh primo" << endl;
	}
}

File: python/bc1165.py
count  = int(input())

for c in range(0, count):
    i = 0
    val = int(input())

    for d in range(1, val+1):
        if val % d == 0: i += 1

    if i == 2: print(f"{val} eh primo")
    else: print(f"{val} nao eh primo")